dev/build/windows/makecoq_mingw.sh

```diff
@@ -2022,7 +2021,7 @@ fi
 
 list_files ocaml_coq
 
-make_addons
+#make_addons
 
 list_files_always ocaml_coq_addons
```

Comment:
For some reason, the job has failed again, despite my commenting of this `make_addons` line. Should I also comment the next one? Anyone who knows, feel free to push (I don't know when I'll be available to do it).

Reply:
@Zimmi48 : sorry, but I don't have the bandwidth to take care of the platform build and the legacy build at the same time. I would say that bumping the OCaml version is larger/more risky change than using the platform build scripts, so I would say we should use the platform installer build scripts. There are only minor issues left (style files missing and the like). But I am open for discussions on this. If you prefer to fix the legacy installer, I can look into that first.

Reply:
No, I prefer you spend time on advancing the platform scripts so that we can ditch the legacy scripts. This was intended as a quick fix for CI, not anything else.